src/autocomplete_test.js

```diff
@@ -110,28 +117,93 @@ module.exports = {
         editor.moveCursorTo(0, 6);
         sendKey("w");
         var popup = editor.completer.popup;
-        check(function () {
+        afterRenderCheck(popup, function () {
             assert.equal(popup.data.length, 1);
             editor.onCommandKey(null, 0, 13);
             assert.equal(popup.data.length, 2);
             assert.equal(editor.getValue(), "goodwill: ");
-            check(function () {
+            afterRenderCheck(popup, function () {
                 editor.onCommandKey(null, 0, 13);
                 assert.equal(editor.getValue(), "goodwill-here");
                 editor.destroy();
                 editor.container.remove();
                 done();
             });
         });
+    },
+    "test: correct completion replacement range when filterText has more than one letter": function (done) {
```

Comment:
where is `filterText` defined?

Reply:
It's not defined, bad test naming, corrected it to reflect what actually is happening